src/app/register-donor/register-donor.component.ts

```diff
@@ -168,8 +191,15 @@ class donorForm {
         }, context);
       }
     }
-    await addDelivery('מחשב', this.computer.value, this.selfDeliver.value);
-    await addDelivery('לפטופ', this.laptop.value, this.selfDeliver.value);
+    if (this.computerAge.value.id == 0) 
```

Comment:
should be:
this.computerAge.value==EquipmentAge.NewEq

Reply:
Alternatively - you can change the code at 103 - to be constructor(public id:number,public caption:string, public isNew:boolean)
and then this could be:
this.computer.Age.value.isNew

Reply:
done